imagetest/test_suites/metadata/startup_shutdown_script_test.go

```diff
@@ -0,0 +1,127 @@
+//go:build cit
+// +build cit
+
+package metadata
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+// The designed shutdown limit is 90s. Let's verify it's executed no less than 80s.
+const shutdownTime = 80
+
+// TestStartupShutdownScripts tests that all startup/shutdown scripts
+// appropriate for the platform ran successfully.
+func TestStartupShutdownScripts(t *testing.T) {
+	isWindows := runtime.GOOS == "windows"
+	for _, ms := range metadataScripts {
+		if ms.windows != isWindows {
+			continue
+		}
```

Comment:
if you accept my suggestion for maps this would obviate the need for this per-key check, instead you would decide which map to iterate over

Reply:
Done